L1Trigger/DTTriggerPhase2/src/MuonPathAssociator.cc

```diff
@@ -133,7 +133,7 @@ void MuonPathAssociator::correlateMPaths(edm::Handle<DTDigiCollection> dtdigis,
         int sl1 = 0;
         int sl3 = 0;
         for (auto SL1metaPrimitive = SL1metaPrimitives.begin(); SL1metaPrimitive != SL1metaPrimitives.end();
-             ++SL1metaPrimitive, sl1++, sl3 = 0) {
+             ++SL1metaPrimitive, sl1++, sl3 = -1) {
```

Comment:
Besides assigning twice a value to `sl3,` I am puzzled; does it remain fixed at "'1"?
And the vector element at [L252](https://github.com/cms-sw/cmssw/blob/752f9ba0c817aa1d279d26949e93878282bdff67/L1Trigger/DTTriggerPhase2/src/MuonPathAssociator.cc#L252) is filled for an index "-1"?